numpy/core/numeric.py

```diff
@@ -3008,6 +3009,11 @@ class errstate(object):
     the context the error handling is set with `seterr` and `seterrcall`, and
     upon exiting it is reset to what it was before.
 
+    ..  versionchanged:: 1.14.0
+        errstate is also usable as a function decorator, saving
+        a level of indentation if an entire function is wrapped.
+        See :py:class:`contextlib.ContextDecorator` for more information. 
```

Comment:
I think if this is a py3 only feature, we shouldn't emphasize this - simply mentioning that it uses `contextlib.contextmanager`, and so inherits any new features that entails is probably the way to go. Similar for the release notes.

Reply:
Version needs to change to 1.15 here too

Reply:
That's done.